<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Introduction of the antimicrobial resistance project | Portfolio</title>
  <meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Introduction of the antimicrobial resistance project | Portfolio" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Introduction of the antimicrobial resistance project | Portfolio" />
  
  <meta name="twitter:description" content="<p>This is a minimal example of using the bookdown package to write a book.
The HTML output format for this example is bookdown::gitbook,
set in the _output.yml file.</p>" />
  

<meta name="author" content="Dian Dupon" />


<meta name="date" content="2023-08-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="analyzing-mass-spectrometry-data-in-r.html"/>
<link rel="next" href="relational-databases.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Portfolio Dian</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html"><i class="fa fa-check"></i><b>2</b> C. elegans offspring count with different components</a>
<ul>
<li class="chapter" data-level="2.1" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#installing-packages"><i class="fa fa-check"></i><b>2.1</b> Installing packages</a></li>
<li class="chapter" data-level="2.2" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#importing-the-data"><i class="fa fa-check"></i><b>2.2</b> Importing the data</a></li>
<li class="chapter" data-level="2.3" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#scatterplot-of-the-c.-elegans-plate-experiment"><i class="fa fa-check"></i><b>2.3</b> Scatterplot of the C. elegans plate experiment</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#normalizing-the-data"><i class="fa fa-check"></i><b>2.3.1</b> Normalizing the data</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#conclusion"><i class="fa fa-check"></i><b>2.4</b> Conclusion</a></li>
<li class="chapter" data-level="2.5" data-path="c.-elegans-offspring-count-with-different-components.html"><a href="c.-elegans-offspring-count-with-different-components.html#further-analysis"><i class="fa fa-check"></i><b>2.5</b> Further analysis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="scoring-a-publication-on-reproducibility.html"><a href="scoring-a-publication-on-reproducibility.html"><i class="fa fa-check"></i><b>3</b> Scoring a publication on reproducibility <br></a>
<ul>
<li class="chapter" data-level="3.1" data-path="scoring-a-publication-on-reproducibility.html"><a href="scoring-a-publication-on-reproducibility.html#information-about-the-study"><i class="fa fa-check"></i><b>3.1</b> Information about the study</a></li>
<li class="chapter" data-level="3.2" data-path="scoring-a-publication-on-reproducibility.html"><a href="scoring-a-publication-on-reproducibility.html#reproducibility-scoring"><i class="fa fa-check"></i><b>3.2</b> Reproducibility scoring</a></li>
<li class="chapter" data-level="3.3" data-path="scoring-a-publication-on-reproducibility.html"><a href="scoring-a-publication-on-reproducibility.html#running-the-open-source-code"><i class="fa fa-check"></i><b>3.3</b> Running the open source code</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="guerrilla-analytics-structure.html"><a href="guerrilla-analytics-structure.html"><i class="fa fa-check"></i><b>4</b> Guerrilla analytics structure</a></li>
<li class="chapter" data-level="5" data-path="curriculum-vitae.html"><a href="curriculum-vitae.html"><i class="fa fa-check"></i><b>5</b> Curriculum vitae</a></li>
<li class="chapter" data-level="6" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html"><i class="fa fa-check"></i><b>6</b> Analyzing mass spectrometry data in R <br></a>
<ul>
<li class="chapter" data-level="6.1" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#introduction-1"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#planning"><i class="fa fa-check"></i><b>6.2</b> Planning</a></li>
<li class="chapter" data-level="6.3" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#data-import"><i class="fa fa-check"></i><b>6.3</b> Data import</a></li>
<li class="chapter" data-level="6.4" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#spectra"><i class="fa fa-check"></i><b>6.4</b> Spectra</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#visualisation-spectra"><i class="fa fa-check"></i><b>6.4.1</b> Visualisation spectra</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#annotation-for-each-spectrum"><i class="fa fa-check"></i><b>6.5</b> Annotation for each spectrum</a></li>
<li class="chapter" data-level="6.6" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#visualization-of-the-data"><i class="fa fa-check"></i><b>6.6</b> Visualization of the data</a></li>
<li class="chapter" data-level="6.7" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#analyzing-maldi-tof-ms-data"><i class="fa fa-check"></i><b>6.7</b> Analyzing MALDI-TOF MS data</a></li>
<li class="chapter" data-level="6.8" data-path="analyzing-mass-spectrometry-data-in-r.html"><a href="analyzing-mass-spectrometry-data-in-r.html#conclusion-1"><i class="fa fa-check"></i><b>6.8</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="introduction-of-the-antimicrobial-resistance-project.html"><a href="introduction-of-the-antimicrobial-resistance-project.html"><i class="fa fa-check"></i><b>7</b> Introduction of the antimicrobial resistance project</a>
<ul>
<li class="chapter" data-level="7.1" data-path="introduction-of-the-antimicrobial-resistance-project.html"><a href="introduction-of-the-antimicrobial-resistance-project.html#antimicrobial-resistance"><i class="fa fa-check"></i><b>7.1</b> Antimicrobial resistance</a></li>
<li class="chapter" data-level="7.2" data-path="introduction-of-the-antimicrobial-resistance-project.html"><a href="introduction-of-the-antimicrobial-resistance-project.html#bacterial-conjugation"><i class="fa fa-check"></i><b>7.2</b> Bacterial conjugation</a></li>
<li class="chapter" data-level="7.3" data-path="introduction-of-the-antimicrobial-resistance-project.html"><a href="introduction-of-the-antimicrobial-resistance-project.html#illumina-paired-end-sequencing"><i class="fa fa-check"></i><b>7.3</b> Illumina paired-end sequencing</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="relational-databases.html"><a href="relational-databases.html"><i class="fa fa-check"></i><b>8</b> Relational databases</a>
<ul>
<li class="chapter" data-level="8.1" data-path="relational-databases.html"><a href="relational-databases.html#introduction-2"><i class="fa fa-check"></i><b>8.1</b> Introduction</a></li>
<li class="chapter" data-level="8.2" data-path="relational-databases.html"><a href="relational-databases.html#required-packages"><i class="fa fa-check"></i><b>8.2</b> Required packages</a></li>
<li class="chapter" data-level="8.3" data-path="relational-databases.html"><a href="relational-databases.html#tidy-data"><i class="fa fa-check"></i><b>8.3</b> Tidy data</a></li>
<li class="chapter" data-level="8.4" data-path="relational-databases.html"><a href="relational-databases.html#inspecting-the-data-with-dbeaver"><i class="fa fa-check"></i><b>8.4</b> Inspecting the data with DBeaver</a></li>
<li class="chapter" data-level="8.5" data-path="relational-databases.html"><a href="relational-databases.html#inspecting-the-data-with-dplyr"><i class="fa fa-check"></i><b>8.5</b> Inspecting the data with dplyr</a></li>
<li class="chapter" data-level="8.6" data-path="relational-databases.html"><a href="relational-databases.html#data-visualisation"><i class="fa fa-check"></i><b>8.6</b> Data visualisation</a></li>
<li class="chapter" data-level="8.7" data-path="relational-databases.html"><a href="relational-databases.html#conclusion-2"><i class="fa fa-check"></i><b>8.7</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="parameterized-report-for-the-covid-19-cases.html"><a href="parameterized-report-for-the-covid-19-cases.html"><i class="fa fa-check"></i><b>9</b> Parameterized report for the COVID-19 cases</a>
<ul>
<li class="chapter" data-level="9.1" data-path="parameterized-report-for-the-covid-19-cases.html"><a href="parameterized-report-for-the-covid-19-cases.html#introduction-3"><i class="fa fa-check"></i><b>9.1</b> Introduction</a></li>
<li class="chapter" data-level="9.2" data-path="parameterized-report-for-the-covid-19-cases.html"><a href="parameterized-report-for-the-covid-19-cases.html#required-packages-1"><i class="fa fa-check"></i><b>9.2</b> Required packages</a></li>
<li class="chapter" data-level="9.3" data-path="parameterized-report-for-the-covid-19-cases.html"><a href="parameterized-report-for-the-covid-19-cases.html#importing-and-filtering-the-data"><i class="fa fa-check"></i><b>9.3</b> Importing and filtering the data</a></li>
<li class="chapter" data-level="9.4" data-path="parameterized-report-for-the-covid-19-cases.html"><a href="parameterized-report-for-the-covid-19-cases.html#data-visualisation-1"><i class="fa fa-check"></i><b>9.4</b> Data visualisation</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction-of-the-antimicrobial-resistance-project" class="section level1 hasAnchor" number="7">
<h1><span class="header-section-number">Chapter 7</span> Introduction of the antimicrobial resistance project<a href="introduction-of-the-antimicrobial-resistance-project.html#introduction-of-the-antimicrobial-resistance-project" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>At the moment I am following the course Data Science for Biology 2 at the Hogeschool Utrecht. In addition of this course we need to fulfill a project in groups of 2 or 3 students. My group got the subject Antimicrobial resistance (AMR). The aim of this project is to map the distribution of AMR genes (which are present on bacterial plasmids) in the Netherlands. This using illumina paired-end sequencing data. This data comes from the SRA database: <a href="https://www.ncbi.nlm.nih.gov/sra" class="uri">https://www.ncbi.nlm.nih.gov/sra</a> <br></p>
<p>To gather more information about this subject it is required to write an introduction for our project. In the text below I wrote a introduction for our project, with the main focus on the following questions: <br>
- What is AMR? <br>
- How do bacteria spread their resistance genes? <br>
- What is conjugation? <br>
- What is illumina sequencing? <br></p>
<div id="antimicrobial-resistance" class="section level2 hasAnchor" number="7.1">
<h2><span class="header-section-number">7.1</span> Antimicrobial resistance<a href="introduction-of-the-antimicrobial-resistance-project.html#antimicrobial-resistance" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Antimicrobial resistance is the ability of micro-organisms to resist antimicrobial treatments. Bacteria wich are resistant to at least three different classes of antimicrobials, defined as multidrug resistant (MDR), have become commonplace. This is especially the case in hospitals. <br></p>
<p>Once a single bacterium mutates to become resistant to antibiotics, it can transfer that resistance to other bacteria around it through a process known as horizontal gene transfer. One of the main vehicles for gene transfer among bacteria are small circular pieces of DNA, or plasmids. Plasmids can be transferred through direct physical contact between bacteria in a process known as conjugation, which helps bacteria share their antibiotic resistance genes with their neighbors. <span class="citation">(<a href="#ref-PlasmidsSpreadAntibiotic" role="doc-biblioref"><span>“Plasmids and the <span>Spread</span> of <span>Antibiotic Resistance Genes</span>”</span> n.d.</a>)</span> <br></p>
</div>
<div id="bacterial-conjugation" class="section level2 hasAnchor" number="7.2">
<h2><span class="header-section-number">7.2</span> Bacterial conjugation<a href="introduction-of-the-antimicrobial-resistance-project.html#bacterial-conjugation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Bacterial conjugation is one of the three major known modes of genetic exchange between bacteria, the other two being transduction and bacterial transformation.<span class="citation">(<a href="#ref-raleighConjugation2013" role="doc-biblioref">Raleigh and Low 2013</a>)</span> <br>
Conjugation is the transfer of a plasmid or other self-transmissible DNA element and sometimes chromosomal DNA from a donor cell to a recipient cell via direct contact usually mediated by a conjugation pilus or sex pilus. <span class="citation">(<a href="#ref-llosaBacterialConjugationTwostep2002" role="doc-biblioref">Llosa et al. 2002</a>)</span> <br></p>
</div>
<div id="illumina-paired-end-sequencing" class="section level2 hasAnchor" number="7.3">
<h2><span class="header-section-number">7.3</span> Illumina paired-end sequencing<a href="introduction-of-the-antimicrobial-resistance-project.html#illumina-paired-end-sequencing" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>For this project illumina paired-end sequencing data was gathered to map the information about the distribution of AMR genes in the Netherlands. The workflow of illumina next generation sequencing can be divided into four steps. This includes library preparation, cluster generation, sequencing and alignment/ data-analysis. <span class="citation">(<a href="#ref-NextGenerationSequencingBeginners" role="doc-biblioref"><span>“Next-<span>Generation Sequencing</span> for <span>Beginners</span> | <span>NGS</span> Basics for Researchers”</span> n.d.</a>)</span> <br></p>
<p>First the DNA is fragmented using ultrasonic fragmentation. This produces fragments of 200-500 bp in lenght. The 5’ and 3’ adapter are added to the two ends of these small segments. <br>
The DNA fragments in the sequencing library will randomly attach to the lanes on the surface of the flow cell when they pass through it. Flow cell is a channel for adsorbing mobile DNA fragments, and it’s also a core sequencing reactor vessel. Each flow cell has 8 Lanes, each lane has a number of adapters attached to the surface, which can match the adapters added at the ends of the DNA fragment in the building process. Bridge PCR was performed using the adapters on flow cell surface as template. After continuous amplification and mutation cycles, each DNA fragment will eventually be clustered in bundles at their respective locations. <br>
The sequencing method is based on sequencing-by-synthesis (SBS). DNA polymerase, connector primers and 4 dNTP with base-specific fluorescent markers are added to the reaction system. The 3′-OH of these dNTP are protected by chemical methods, which ensures that only one base will be added at a time during the sequencing process. All unused free dNTP and DNA polymerase are eluted after the synthesis reaction finished.
Then, buffer solution needed for fluorescence excitation are added, the fluorescence signal is excited by laser, and fluorescence signal is recorded by optical equipment. Finally, the optical signal is converted into sequencing base by computer analysis. When the fluorescence signal is recorded, a chemical reagent is added to quench the fluorescence signal and remove the dNTP 3′-OH protective group, so that the next round of sequencing reaction can be performed. The newly identified sequence reads can be aligned to a reference genome. <span class="citation">(<a href="#ref-cdadminPrincipleWorkflowIllumina2018" role="doc-biblioref">cdadmin 2018</a>)</span> <br></p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-cdadminPrincipleWorkflowIllumina2018" class="csl-entry">
cdadmin. 2018. <span>“Principle and <span>Workflow</span> of <span class="nocase">Illumina Next-generation Sequencing</span> | <span>CD Genomics Blog</span>.”</span> October 17, 2018. <a href="https://www.cd-genomics.com/blog/principle-and-workflow-of-illumina-next-generation-sequencing/">https://www.cd-genomics.com/blog/principle-and-workflow-of-illumina-next-generation-sequencing/</a>.
</div>
<div id="ref-llosaBacterialConjugationTwostep2002" class="csl-entry">
Llosa, Matxalen, F. Xavier Gomis-Rüth, Miquel Coll, and Fernando de la Cruz Fd. 2002. <span>“Bacterial Conjugation: A Two-Step Mechanism for <span>DNA</span> Transport.”</span> <em>Molecular Microbiology</em> 45 (1): 1–8. <a href="https://doi.org/10.1046/j.1365-2958.2002.03014.x">https://doi.org/10.1046/j.1365-2958.2002.03014.x</a>.
</div>
<div id="ref-NextGenerationSequencingBeginners" class="csl-entry">
<span>“Next-<span>Generation Sequencing</span> for <span>Beginners</span> | <span>NGS</span> Basics for Researchers.”</span> n.d. Accessed May 24, 2023. <a href="https://www.illumina.com/science/technology/next-generation-sequencing/beginners.html">https://www.illumina.com/science/technology/next-generation-sequencing/beginners.html</a>.
</div>
<div id="ref-PlasmidsSpreadAntibiotic" class="csl-entry">
<span>“Plasmids and the <span>Spread</span> of <span>Antibiotic Resistance Genes</span>.”</span> n.d. <span>ASM.org</span>. Accessed May 24, 2023. <a href="https://asm.org:443/Articles/2023/January/Plasmids-and-the-Spread-of-Antibiotic-Resistance-G">https://asm.org:443/Articles/2023/January/Plasmids-and-the-Spread-of-Antibiotic-Resistance-G</a>.
</div>
<div id="ref-raleighConjugation2013" class="csl-entry">
Raleigh, E. A., and K. B. Low. 2013. <span>“Conjugation.”</span> In <em>Brenner’s <span>Encyclopedia</span> of <span>Genetics</span> (<span>Second Edition</span>)</em>, edited by Stanley Maloy and Kelly Hughes, 144–51. <span>San Diego</span>: <span>Academic Press</span>. <a href="https://doi.org/10.1016/B978-0-12-374984-0.00321-1">https://doi.org/10.1016/B978-0-12-374984-0.00321-1</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="analyzing-mass-spectrometry-data-in-r.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="relational-databases.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/06-Introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Dian-portfolio.pdf", "Dian-portfolio.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
